lib/widgets/drive.coffee

```diff
@@ -51,18 +50,34 @@ getDrives = ->
 # visuals.drive('Please select a drive').then (drive) ->
 # 	console.log(drive)
 ###
-module.exports = (message = 'Select a drive') ->
-	getDrives().then (drives) ->
 
-		if _.isEmpty(drives)
-			throw new Error('No available drives')
+module.exports = (message = 'Select a drive') ->
 
-		form.ask
-			type: 'list'
-			name: 'drive'
+	getDrives().then (drives) ->
+		options =
 			message: message
+			name: 'list'
 			choices: _.map drives, (drive) ->
 				return {
 					name: "#{drive.device} (#{drive.size}) - #{drive.description}"
```

Comment:
This string interpolation is being duplicated twice below. Please refactor to remove the duplication.